page-home.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/*
+Template Name: Homepage
+*/
+get_header(); ?>
+
+<main id="primary" role="main" class="content-area">
+	<?php
+		global $post;
+		$status = get_post_meta( $post->ID, 'home_banner_status', true );
+		$image = get_post_meta( $post->ID, 'home_banner_img', true );
+		$title = get_post_meta( $post->ID, 'home_banner_title', true );
+		$excerpt = get_post_meta( $post->ID, 'home_banner_excerpt', true );
+		$url = get_post_meta( $post->ID, 'home_banner_url', true );
+		$button = get_post_meta( $post->ID, 'home_banner_btn', true );
+
+		echo home_banner( $status, $image, $title, $excerpt, $url, $button );
+	?>
+	<div class="container">
+		<div class="row">
+			<div class="card-wrapper equal-heights-flex-box clearfix">
+				<?php
+
+				$transient = get_transient( 'homepage_cards_html' );
```

Comment:
Yep, I'm planning to do this for the next iteration, as I may use transient for the home banner as well.